broad/broad/spiders/broad.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import re
+import datetime
+
+from six.moves.urllib.parse import urlparse
+import click
+import scrapy
+from scrapy.http import Request, HtmlResponse
+from scrapy.linkextractors import LinkExtractor
+
+from broad.items import Page
+
+
+class BroadSpider(scrapy.Spider):
+    name = "broad"
+    ratings_map = {
+        'one': 1,
+        'two': 2,
+        'three': 3,
+        'four': 4,
+        'five': 5,
+    }
+
+    with open('domains.txt') as f:
+        start_urls = [url.strip() for url in f.readlines()]
```

Comment:
yes - so instead of ``start_urls = [url.strip() for url in f.readlines()]`` I meant something like ``start_urls = ['http://domain{}:{}'.format(i, port) for i in range(1, n_domains + 1)]``, and getting rid of the file. This way it's easier to change number of domains and port.